python/lsst/ctrl/stats/records/evicted.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, year, lines):
         # bytes received during the run
         self.runBytesReceived = int(self.extract(pat, lines[5], "bytes"))
 
-        pat = r"Partitionable Resources :\s+Usage\s+\Request\s+Allocated$"
+        pat = r"Partitionable Resources :\s+Usage\s+\s+Request\s+Allocated$"
```

Comment:
This was actually a bug;  should have just been \s+ instead.   I was focused on the \R part, instead of looking what was happening before that.